client/src/components/Navigation.js

```diff
@@ -19,6 +30,11 @@ export default function Navigation({ isStatic }) {
   );
 }
 
+const BackIcon = styled(UnfoldIcon)`
+  margin: 1.4rem 0;
+  transform: scale(1.5) rotate(90deg);
+`;
```

Comment:
^ –> < ? :)

Reply:
Had to think a little about the meaning of this comment 😆 but yes, that's right!